### DISCUSSION TITLE
Showing full width figures in `reference-hover` regardless of the `width` parameter.

### DISCUSSION THREAD

**Discussion by apcamargo** [quarto-dev/quarto-cli#9888] [Q&A] [answered]
### Description

When I reduce the width of a figure so it doesn't fill the entire width, the `reference-hover` is also affected:

<img width="672" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/22940964/c29144ac-a480-4277-b0f9-5689d3777e8f">

````qmd
See @fig-1 for an example.

![A Lovely Image](images/mv-0.jpg){#fig-1 width=75%}
````

Is there any way to prevent the hover preview from having this white space?

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us  to recreate your problem so that we can fix it: please help us help you! Thanks.

This being said, hover is HTML based so you can use (S)CSS and your favourite browser inspect/developer mode.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by apcamargo** [comment, marked as answer]
Thank you! I fixed this using the following CSS rule:

```css
.tippy-content img {
    width: 100% !important;
}
```

**Reply by apcamargo** [reply]
@mcanouil @gordonwoodhull do you think this is something that deserves an issue? My feeling is that the current behaviour is not what users would expect (on top of not being aestheticaly pleasing). In my opinion, images should be displayed across the full width of the tooltip. Not sure if this is something that should be discussed here or in an issue.

**Reply by mcanouil** [reply]
This would need to be tested in more scenarios to ensure it does not lead to weirder things than the current white space.

**Reply by mcanouil** [reply]
I've opened an issue to track this as a future enhancement:
- #9950
